lib/openshift/Console_CLI_Download.rb

```diff
@@ -0,0 +1,8 @@
+require 'openshift/cluster_resource'
+
+module BushSlicer
+  # @note represents an OpenShift cluster resource about console
+  class ConsoleCliDownload < ClusterResource
+    RESOURCE = 'consoleclidownload'
```

Comment:
is this the actual API name? What does `oc get -o yaml consoleclidownload` return? See example `config_samples_operator_openshift_io.rb` for a more recent resource type.

Reply:
@akostadinov consoleclidownload indeed is an API and `oc get consoleclidownload` returns `No resources found` if haven't yet been created.

Reply:
Good to merge? Some cases are waiting for `check_link_and_text` to be reliable :)

Reply:
Could you paste `-o yaml` output from the object? The string as I tried to explain above doesn't look like a full API name.

Reply:
@akostadinov 
```
$ oc get consoleclidownload example -o yaml
apiVersion: console.openshift.io/v1
kind: ConsoleCLIDownload
metadata:
  creationTimestamp: "2019-10-28T07:17:24Z"
  generation: 1
  name: example
  resourceVersion: "115272"
  selfLink: /apis/console.openshift.io/v1/consoleclidownloads/example
  uid: fd571ac2-f952-11e9-b132-02c218734c54
spec:
  description: |
    This is an example CLI download description that can include markdown such as paragraphs, unordered lists, code, [links](https://www.example.com), etc.

    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce a lobortis justo, eu suscipit purus.
  displayName: examplecli
  links:
  - href: https://www.example.com/
```

Reply:
@akostadinov Also I can `oc explain` the resource
```
$ oc explain consoleclidownload
KIND:     ConsoleCLIDownload
VERSION:  console.openshift.io/v1

DESCRIPTION:
     <empty>

FIELDS:
   apiVersion	<string>
     APIVersion defines the versioned schema of this representation of an
     object. Servers should convert recognized schemas to the latest internal
     value, and may reject unrecognized values. More info:
     https://git.k8s.io/community/contributors/devel/api-conventions.md#resources

   kind	<string>
     Kind is a string value representing the REST resource this object
     represents. Servers may infer this from the endpoint the client submits
     requests to. Cannot be updated. In CamelCase. More info:
     https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds

   metadata	<Object>
     Standard object's metadata. More info:
     https://git.k8s.io/community/contributors/devel/api-conventions.md#metadata

   spec	<Object>
     Represents console CLI download spec
```

Reply:
looks like it is `consoleclidownloads.console.openshift.io`

Reply:
@akostadinov So, you mean it should be `RESOURCE = 'onsoleclidownloads.console.openshift.io'` ?

Reply:
yes, also filename better to match. Like `config_samples_operator_openshift_io.rb`